roles/kubernetes-apps/ansible/tasks/cleanup_dns.yml

```diff
@@ -1,31 +1,4 @@
 ---
-- name: Kubernetes Apps | Delete old CoreDNS resources
```

Comment:
We need to make sure kubeadm does not deploy coredns then

Reply:
Actually it's taken care of in https://github.com/kubernetes-sigs/kubespray/pull/4317

We can skip coredns stage now.

Reply:
We should wait for #4317 to get merged first before we merge this :)